lib/server.js

```diff
@@ -186,43 +186,9 @@ function ifError(n) {
 
 
 /**
- * when an error occurrs, this is used to emit an error to consumers
- * via EventEmitter.
- * @private
- * @function emitRouteError
- * @param    {Object} server the server object
- * @param    {Object} req    the request object
- * @param    {Object} res    the response object
- * @param    {Object} err    an error object
- * @returns  {undefined}
- */
-function emitRouteError(server, req, res, err) {
-
-    var name;
-
-    if (err.name === 'ResourceNotFoundError') {
-        name = 'NotFound';
-    } else if (err.name === 'InvalidVersionError') {
-        name = 'VersionNotAllowed';
-    } else {
-        name = err.name.replace(/Error$/, '');
-    }
-
-    req.log.trace({name: name, err: err}, 'entering emitRouteError');
-
-    if (server.listeners(name).length > 0) {
-        server.emit(name, req, res, err, once(function () {
-            res.send(err);
-            server._finishReqResCycle(req, res, null, err);
-        }));
-    } else {
-        res.send(err);
-        server._finishReqResCycle(req, res, null, err);
-    }
-}
-
-
-/**
+ * TODO: this is relevant for CORS only. Should move this out eventually to a
```

Comment:
👍  on this idea! I think it might be useful to have a new label then, something like.. `internal` for these non user facing clean up tasks/refactoring.